rllib/tuned_examples/ppo/multi_agent_stateless_cartpole_ppo.py

```diff
@@ -0,0 +1,69 @@
+from ray.rllib.algorithms.ppo import PPOConfig
+from ray.rllib.connectors.env_to_module import MeanStdFilter
+from ray.rllib.examples.envs.classes.multi_agent import MultiAgentStatelessCartPole
+from ray.rllib.utils.metrics import (
+    ENV_RUNNER_RESULTS,
+    EPISODE_RETURN_MEAN,
+    NUM_ENV_STEPS_SAMPLED_LIFETIME,
+)
+from ray.rllib.utils.test_utils import add_rllib_example_script_args
+from ray.tune import register_env
+
+parser = add_rllib_example_script_args(default_timesteps=4000000)
+parser.set_defaults(
+    enable_new_api_stack=True,
+    num_agents=2,
+    num_env_runners=3,
+)
+# Use `parser` to add your own custom command line options to this script
+# and (if needed) use their values toset up `config` below.
+args = parser.parse_args()
+
+register_env(
+    "multi_stateless_cart",
+    lambda _: MultiAgentStatelessCartPole({"num_agents": args.num_agents}),
+)
+
+config = (
+    PPOConfig()
+    # Enable new API stack and use EnvRunner.
+    .api_stack(
+        enable_rl_module_and_learner=True,
+        enable_env_runner_and_connector_v2=True,
+    )
+    .environment("multi_stateless_cart")
+    .env_runners(
+        env_to_module_connector=lambda env: MeanStdFilter(multi_agent=True),
```

Comment:
I remember, we have still this open question, if we need to add this connector also to learner or not. I think we do not need it:  MeanStd rewrites observations and learner has these observations then, correct?

Reply:
Correct, this connector (and most other env-to-module ones) directly writes back into the episode, thus making the change to the observation permanent. Hence no need to also add it to the Learner pipeline as the Learner pipeline then operates on the already changed episodes/observations.